airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -37,16 +41,54 @@ fi
 echo "Mounting data partition"
 mount /dev/sda3 /mnt
 
-echo "Decompressing and writing:"
-lz4 -c -d /mnt/image.img.lz4 | pv -s 50g > /dev/nvme0n1 
-#lz4 -c -d /usr/share/vx-img/image.img.lz4 | pv -s 50g > /dev/nvme0n1 
+_path="/mnt"
+_supported=('gz' 'lz4')
+_hashash=0
+
+_toflash=""
+for f in $_path/*; do
+    _filename="${f##*/}"
```

Comment:
can you explain what "${f##*/}" does just for my own knowledge haha

Reply:
All it does is peel off the leading path, so `/home/foo/bar.gz` to just `bar.gz`